README.md

```diff
@@ -33,43 +36,56 @@ Refinery is designed to sit within your infrastructure where all traces can reac
 
+- a `linux/amd64` or `linux/arm64` operating system
+- 2GB RAM for each server used
+- Access to 2 cores for each server used
+
+In many cases, Refinery only needs one node.
+Refinery clusters usually run best with fewer, larger servers; for large installations, 16 cores and 16GB of RAM per server is not unusual (with appropriate tuning).
+If experiencing a large volume of traffic, you may need to scale out to multiple nodes, and likely need a small Redis cluster to handle scaling.
+
+We recommend increasing the amount of RAM and the number of cores after your initial set-up.
 Additional RAM and CPU can be used by increasing configuration values; in particular, `CacheCapacity` is an important configuration value. Refinery's `Stress Relief` system provides a good indication of how hard Refinery is working, and when invoked, logs (as `reason`) the name of the Refinery configuration value that should be increased to reduce stress.
+Use our [scaling and troubleshooting documentation](https://docs.honeycomb.io/manage-data-volume/refinery/scale-and-troubleshoot/) to learn more.
 
 ### Setting up Refinery in Kubernetes
 
 Refinery is available as a Helm chart in the Honeycomb Helm repository.
 
-You can install Refinery with the following command:
+You can install Refinery with the following command, which uses the default values file:
 
 ```bash
 helm repo add honeycomb https://honeycombio.github.io/helm-charts
 helm install refinery honeycomb/refinery
 ```
 
-This will use the default values file. You can also supply your own:
+Alternatively, supply your own custom values file:
 
 ```bash
 helm install refinery honeycomb/refinery --values /path/to/refinery-values.yaml
 ```
 
+where `/path/to/refinery-values.yaml` is the file's path.
+
 ## Peer Management
 
 When operating in a cluster, Refinery expects to gather all of the spans in a trace onto a single instance so that it can make a trace decision. Since each span arrives independently, each Refinery instance needs to be able to communicate with all of its peers in order to distribute traces to the correct instance.
 
 This communication can be managed in two ways: via an explicit list of peers in the configuration file, or by using self-registration via a shared Redis cache. Installations should generally prefer to use Redis. Even in large installations, the load on the Redis server is quite light, with each instance only making a few requests per minute. A single Redis instance with fractional CPU is usually sufficient.
 
-
 ## Configuration
 
-Configuration is controlled by Refinery's two configuration files. For information on all of the configuration parameters that control Refinery's operation, please see [the configuration documentation](https://docs.honeycomb.io/manage-data-volume/refinery/configuration/).
+Configuration is controlled by Refinery's two configuration files - `config.yaml` for general configuration and `rules.yaml` for sampling configuration.
```

Comment:
I deliberately didn't give them names. I won't stop you, but in the past I think it has caused more confusion than it has helped.

Reply:
@kentquirk How about this instead?
```suggestion
Configuration is controlled by Refinery's two configuration files, which is generally referred to as `config.yaml` for general configuration and `rules.yaml` for sampling configuration.
```